packages/ui-app/src/CopyButton.tsx

```diff
@@ -85,4 +87,8 @@ export default styled(CopyButton)`
   button.ui.mini.icon.primary.button.iconButton {
     cursor: copy;
   }
+
+  .copySpan {
```

Comment:
Does this really make a difference? The span is basically around a single element? (A button with only an icon, which should not be breakable)

Reply:
This is black magic.. Found the trick *on the internet* and it does work on both chrome and FF :man_shrugging: